### DISCUSSION TITLE
How to use directory names with numbers?

### DISCUSSION THREAD

**Discussion by ghost** [datenstrom/community#307] [Ask a question] [answered]
Hi,

I can create a directory with numbers, for example 2014. The directory can also be accessed via the URL. If I now rename the directory, for example 1-2014, it is no longer accessible at the URL.

As I understand it, a number followed by a hyphen is used for sorting and displaying in navigation?

So can not I use numbers for a directory if I want to have it displayed in the navigation?

**Comment by markseuffert** [comment]
Yes you can use folder names with numbers and hyphens, for example `2014-01`. There could be a different problem, because [the first folder folder is responsible for the home page](https://datenstrom.se/yellow/help/how-to-change-the-content). If you rename rename `1-home` to `0-home`, that should put the home page first and solve the naming problem. 

Let me know if that works for you?

**Comment by ghost** [comment]
Hi Mark,

In the case described is my directory structure:

1-home
2-test
and in 2 a subfolder
5-blah
and in 5 a subfolder
1-2014

If the subfolder is created only as 2014, the call is not a problem.

**Comment by markseuffert** [comment, marked as answer]
Hi Jan,

in this case the folder `content/2-test/5-blah/1-2014` becomes `http://website/test/blah/1-2014/`. The rule for a folder name is that a prefix consisting of numbers and hyphens will be removed from the URL, unless the folder name consists only of numbers and hyphens. 

Not sure if I could explain this right. Here's something you can do. Log into your website, append the folders to the end of the URL, e.g. `https://developers.datenstrom.se/edit/hello/world/2014/`, click "create this page". Now the necessary folder structure will be created.

**Comment by ghost** [comment]
Hi Mark,

Thank you for your explanation. I do not use the possibility of the edit function, I work with my text editor ;-)

But ... I now create a new folder in the content folder, for example 9-2014, then this is also displayed in the navigation - because he just a number separated from a hyphen - contains. What if I but this folder, which is indeed numeric (2014), not want to have displayed in the navigation? ;-) There, the folder appears anyway, after he was renamed from 9-2014 in 2014.

I noticed this only by accident experimenting and only because of my demand. So are numeric folders, if you need them, always displayed in a navigation?

Greetings - Jan
